session_header.php

```diff
@@ -0,0 +1,7 @@
+<?php
+session_start();
+
+if ( ! isset($_SESSION['emp_id']) && ( ! isset($_SESSION['is_completed']) && 1 != $_SESSION['is_completed']))
```

Comment:
Use === instead of ==
parenthesis around isset is not required
Comments can be added for the functionality